run.php

```diff
@@ -26,8 +26,18 @@
 
 require(dirname(__FILE__) . '/../../config.php');
 require_once($CFG->libdir . '/clilib.php');
-require_once($CFG->dirroot . '/local/codechecker/locallib.php');
 
+// PHP_Codesniffer autoloading.
+if (is_file(__DIR__ . '/phpcs/autoload.php') === true) {
+    include_once(__DIR__ . '/phpcs/autoload.php');
+} else {
+    include_once('PHP/CodeSniffer/autoload.php');
+}
+// PHPCompatibility autoloading.
+require_once('PHPCSAliases.php');
```

Comment:
IIUC, this file is included through _PHPCompatibility/ruleset.xml_, so we don't need to include it here.

Reply:
This part has been copied from https://github.com/moodlehq/moodle-local_codechecker/blob/phpcs3/index.php#L28-L34 so if this needs to be changed, I would suggest doing in a separate issue.

Reply:
No problem. Could you create the issue? Thanks.